proto/daemon/v1alpha/sites.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+package com.mintter.daemon.v1alpha;
+
+import "site/v1alpha/site.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "mintter/backend/genproto/daemon/v1alpha;daemon";
+
+// Sites API allows to configure the remote Mintter Sites inside the local app.
+// To interact with the existing site, the local daemon will expose the Site API
+// interface as well (see /proto/site/v1alpha/site.proto), requiring a gRPC metadata
+// header with the hostname of the actual site to know the endpoint where to proxy the request.
+service Sites {
+  // Adds a site configuration to the local app.
+  rpc AddSite(AddSiteRequest) returns (SiteConfig);
+
+  // Deletes the site configuration from the local app.
+  rpc DeleteSite(DeleteSiteRequest) returns (google.protobuf.Empty);
+
+  // Lists configured sites.
+  rpc ListSites(ListSitesRequest) returns (ListSitesResponse);
+
+  // Given a docuement, lists all the sites it's been published to.
+  rpc GetDocWebPublications(GetDocWebPublicationsRequest) returns (GetDocWebPublicationsResponse);
```

Comment:
This should be an authenticated call, therefore it should go in the site.proto definition, right?